search-relevancy-test/src/search_relevancy_test/anomaly_fetcher.clj

```diff
@@ -35,8 +35,10 @@
   "Returns a map of concept-id to the metadata format for that collection."
   [concept-ids]
   (let [url (str (source-url) "/collections.umm-json")
-        response (client/get url {:query-params {:concept-id concept-ids
+        response (client/post url {:form-params {:concept-id concept-ids
                                                  :page-size 2000}})
+                                  ;; Uncomment when collections are only visible for a certain token
+                                  ;  :headers {:echo-token (System/getenv "CMR_SYSTEM_TOKEN")}})
```

Comment:
Can we just always pass the token, that way you wouldn't have to remember to uncomment it out?

Reply:
Fixed to use a token if configuration is set and not pass in the header otherwise.